packages/circuit-ui/components/Input/Input.tsx

```diff
@@ -55,9 +55,14 @@ export interface InputProps extends Omit<HTMLProps<HTMLInputElement>, 'label'> {
    */
   renderSuffix?: FC<{ className?: string }>;
   /**
-   * Warning or error message, displayed in a tooltip.
+   * Warning or error message, displayed below the input.
    */
   validationHint?: string;
+  /**
+   * Label to indicate that the input is optional. Only displayed when the
+   * `required` prop is falsy.
+   */
+  optionalLabel?: string;
```

Comment:
Should we explicitly make this required when `required` is missing, and `never` when it's there?